src/main/java/duke/tasks/TaskType.java

```diff
@@ -0,0 +1,25 @@
+package duke.tasks;
+
+
+/**
+ * TaskType enum.
```

Comment:
Similar to the other enumeration file you have, you can describe what kind of enumerations are inside